RELEASING.md

```diff
@@ -2,69 +2,85 @@
 
 This document describes how to release this component.
 
+The component is released using GitHub automation.
 
-## Release Pre-Requisites
+At a high level, the process is as follows:
 
-In addition to the build pre-requistes:
+1. The developer adds their PGP private key/passphrase as repository secrets
+1. The workflow `stage_release` tags, builds/signs the release, and stages the release on a Nexus staging repository.
+1. The stage release is verified using manual verification steps.
+1. The workflow `deploy_release` releases from the staged repository to Maven Central.
+1. The developer removes their PGP private key/passphrase as repository secrets.
 
-- [`gh`](https://cli.github.com/) Github CLI (optional)
+## Pre-Requisites
 
-## Releasing this project
+You must be a member of the Kroxylicious organization and have access to [create 
+secrets](https://github.com/kroxylicious/kroxylicious-junit5-extension/settings/secrets/actions) within the
+kroxylicious-junit5-extension repository.  Create the following repository secrets.
 
-Firstly ensure that you have completed the [release requirements](https://github.com/kroxylicious/kroxylicious/blob/main/RELEASING.md#requirements).  Once that is done, use
-the release script to actually perform the release and prepare main for the next development version.
 
-```shell
-./scripts/release.sh -k <YOUR_KEY_SHORT_ID> -v <RELEASE_VERSION>
-```
+| Secret                               | Description                                   |
+|--------------------------------------|-----------------------------------------------|
+| KROXYLICIOUS_RELEASE_GPG_PRIVATE_KEY | GPG private key for Kroxylicious (plain text) |
```

Comment:
I think we should be explicit here that its the key of the *person* running the release and not the project.
```suggestion
| KROXYLICIOUS_RELEASE_GPG_PRIVATE_KEY | GPG private key for signing Kroxylicious (plain text) |
```